blog/mindestlohnerhöhung/index.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: 12 € Mindestlohn und Minijobgrenze erhöht - was Sie wissen müssen.
-publishDate: 2022-10-15
+date: 2022-10-15
 slug: Mindestlohnerhöhung
-cover: "münzstapel.png"
+cover: "münzstapel.webp"
```

Comment:
Gutes Bild übrigens! Wo hast du das gefunden? Bitte bei der Auswahl immer auch auf Copyright achten! https://unsplash.com ist eine super Quelle für Copyright-sichere bilder.

Reply:
Ach ja und das Bild war viel zu groß (> 2mb!). Bitte in Zukunft Bilder immer erst zurechtschneiden bzw umskalieren. In der Breite muss ein Bild eigentlich nie größer als 2000 px sein. Noch besser wäre es, wenn du das Bild auch noch in das `webp` format umwandeln könntest. Google vielleicht mal, wie das geht und ob du das hinkriegst.

Reply:
Von unsplash habe ich das Bild auch - habe explizit nach Copyright freien Bildern gesucht